client/components/data/query-memberships-subscriptions/index.js

```diff
@@ -0,0 +1,35 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import isRequestingMembershipsSubscriptions from 'state/selectors/is-requesting-memberships-subscriptions';
```

Comment:
we shouldn't be building this into state. it's already provided if we use request tracking or if we use `httpData`

Reply:
Done as you propose in https://github.com/Automattic/wp-calypso/pull/26023